### PR TITLE
sed with -i option now runs silently

### PR DESCRIPTION
**Pull request by joshi-sh**
Fixes [issue #159](https://github.com/shelljs/shx/issues/159) with shx

### CODE REVIEW

File: test/sed.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ test('glob file names, with in-place-replacement', t => {
   const result = shell.sed('-i', 'test', 'hello', `${t.context.tmp}/file*.txt`);
   t.falsy(shell.error());
   t.is(result.code, 0);
-  t.is(result.toString(), 'hello1\n\nhello2\n'); // TODO: fix sed's behavior
```
Comment (nfischer) [inline test/sed.js]: I think this TODO is about the double newline between `hello1` and `hello2`, not the `-i` behavior. Could you file a github issue to track the double-newline thing?

**Comment by zone117x** [comment]
Hi all, is there anything preventing this PR from merging?

**Comment by catchmareck** [comment]
I need that too :/ could you please merge this?

File: test/sed.js (hunk 1 above)
Comment (nfischer) [reply test/sed.js:173]: https://github.com/shelljs/shelljs/issues/1068

**Comment by pringshia** [comment]
Any plans on when the next release might be? Would be great to get this into `shx`.

**Pull request merged**